### ISSUE TITLE
[a11y] Audit Alert

### ISSUE THREAD

**Issue by samuel-gomez-axa** [AxaFrance/react-toolkit#1141]
# Vérification de l'accessibilité du composant : Alert

Afin d'évaluer le niveau d'accessibilité d'un composant, il faut effectuer les actions ci-dessous.

## Analyse statique d'accessibilité

Remplir les champs ci-dessous, en se basant sur l'onglet accessibilité de chaque composant dans le storybook.

**Alert danger :**
![image](https://github.com/AxaFrance/react-toolkit/assets/1991349/97c45621-6203-492c-83ac-cd166e3dc917)

**VIOLATIONS** : 1 : contraste de couleur
*INCOMPLETE* : 0
PASSED : 3

**Alert success :**
![image](https://github.com/AxaFrance/react-toolkit/assets/1991349/67c64386-db1d-44d9-b31c-31bce5de2eab)

**VIOLATIONS** : 0
*INCOMPLETE* : 0
PASSED : 4

**Alert info :**
![image](https://github.com/AxaFrance/react-toolkit/assets/1991349/3c17d73e-f462-4256-b191-dddb0989628d)

**VIOLATIONS** : 1 : contraste de couleur
*INCOMPLETE* : 0
PASSED : 3

**Alert Error :**
![image](https://github.com/AxaFrance/react-toolkit/assets/1991349/63596324-a2ae-4703-b73f-1478c3c188c6)

**VIOLATIONS** : 0
*INCOMPLETE* : 0
PASSED : 4

**Alert avec bouton de fermeture :**
![image](https://github.com/AxaFrance/react-toolkit/assets/1991349/83329a36-6dd9-4bd7-bb8b-bb1e9ed35092)

**VIOLATIONS** : 1 : le bouton de fermeture n'a pas de texte
*INCOMPLETE* : 0
PASSED : 6

**Alert avec détails :**
![image](https://github.com/AxaFrance/react-toolkit/assets/1991349/40691ad6-d938-4106-8dc1-28682ea0cd6e)

**VIOLATIONS** : 0
*INCOMPLETE* : 0
PASSED : 7

En plus mettre une capture de l'onglet en commentaire de cette issue.

## Analyse manuelle

- [x] Est ce que le composant est navigable au clavier (tab) ?
Composant non navigable à l'exception du bouton de fermeture qui est bien navigable au clavier.
- [ ] Est ce que le composant est navigable avec NVDA ?
![image](https://github.com/AxaFrance/react-toolkit/assets/1991349/49de0fc8-9e5e-4f89-b0cf-42ce172b1d41)
Lorsque l'alerte apparait à l'écran elle est bien lue par NVDA via le `role="alert"`. Néanmoins pour l'alerte avec détails, seul le titre est lu, la liste de détails n'est pas lue.
- [ ] Est ce que le composant est navigable avec un autre lecteur d'écran ?

## Vérification sémantique

- [x] Vérifier l'implémentation sémantique des éléments html ?
- [x] Vérifier la validité HTML W3C (https://validator.w3.org/#validate_by_input) ?
- [x] Vérifier la validité CSS (https://jigsaw.w3.org/css-validator/#validate_by_input) ?

## Problèmes détectés

Si vous avez détectés un défaut d'accessibilité, veuillez créer une issue décrivant le défaut et la rattacher à cette issue.

**Comment by MartinWeb** [comment]
Problèmes rencontrés :

- Contraste de couleur
- Bouton de fermeture de l'alerte sans texte discernable
- Texte de l'alerte avec détails pas complètement lu par NVDA

**Comment by MartinWeb** [comment]
Report sur le projet design system
